src/datasets/commands/datasets_cli.py

```diff
@@ -5,7 +5,7 @@
 from datasets.commands.dummy_data import DummyDataCommand
 from datasets.commands.env import EnvironmentCommand
 from datasets.commands.run_beam import RunBeamCommand
-from datasets.commands.test import TestCommand
+from datasets.commands.test import CLITestCommand
```

Comment:
You are preceding with `CLI` only the `TestCommand`; i.e. the rest of the command are not preceded by `CLI`.

IMHO, either all or none should be preceded by `CLI`.

On the other hand, from your commit message, I guess you introduce this modification to prevent `pytest` default test discovery to identify this class as a test. If this is the reason, I would strongly suggest using `testpaths` instead: https://docs.pytest.org/en/latest/reference/reference.html#confval-testpaths
- We tell `pytest` to search for tests only inside the `tests` directory
- Moreover, tests discovery/collection will be faster

Reply:
`pytest` suggest adding the class attribute `__test__ = False` to `TestCommand`, but I am not sure if this is better... :/